modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordWriterTrackerImpl.java

```diff
@@ -51,50 +52,76 @@ public Map<String, String> getDDMFormInstanceRecordWriterExtensions() {
 			_ddmFormInstanceRecordWriterExtensions);
 	}
 
-	@Reference(
-		cardinality = ReferenceCardinality.MULTIPLE,
-		policy = ReferencePolicy.DYNAMIC,
-		policyOption = ReferencePolicyOption.GREEDY
-	)
-	protected void addDDMFormInstanceRecordWriter(
-		DDMFormInstanceRecordWriter ddmFormInstanceRecordWriter,
-		Map<String, Object> properties) {
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		_serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
```

Comment:
@JanisZhang Since you still need to add a customizer and populate another map, I think it's better to make it a normal `ServiceTracker` and populate both maps in the customizer. Please modify and resend.